### PR TITLE
Tickets: Use project instead of namespace

### PR DESCRIPTION
**Pull request by petersutter**
**What this PR does / why we need it**:
Tickets, that are now created will have the project name in the title. Hence also the mapping from ticket to namespace / shoot had to be adapted accordingly.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement user

```

### CODE REVIEW

File: frontend/src/components/ShootTickets/TicketComment.vue [hunk 1]
```diff
@@ -94,6 +94,30 @@ export default {
     word-wrap: break-word;
     /* does not work with firefox */
     word-break: break-word;
+
```
Comment (grolu) [inline frontend/src/components/ShootTickets/TicketComment.vue:97]: why is this css now required?

Comment (petersutter) [reply frontend/src/components/ShootTickets/TicketComment.vue:97]: it's not required, however I overwrote the font-size of the headings and adapted them to the github sizes. Previously, I felt they were a bit too large

Comment (grolu) [reply frontend/src/components/ShootTickets/TicketComment.vue:97]: okay, I just wanted to confirm because it has nothing to do with the rest of the PR

**Pull request merged**